pkg/webhook/policy.go

```diff
@@ -307,6 +312,17 @@ func (h *validationHandler) validateConstraint(ctx context.Context, req admissio
 	return false, nil
 }
 
+func (h *validationHandler) validateConfigResource(ctx context.Context, req admission.Request) (bool, error) {
+	gvk := req.AdmissionRequest.Kind
+	correctGK := gvk.Group == "config.gatekeeper.sh" && gvk.Kind == "Config"
+
+	if correctGK && req.Name != "config" {
```

Comment:
Should we make `config` into a const?

Reply:
I think it may be a const in the config controller. We may be able to import it?

Reply:
Done